static/usage/alert/buttons/angular/angular_ts.md

```diff
@@ -0,0 +1,38 @@
+```ts
+import { Component } from '@angular/core';
+import { AlertController } from '@ionic/angular';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: 'app.component.html',
+})
+export class AppComponent {
+  handlerMessage = "";
```

Comment:
~ Minor nitpick, this and `roleMessage` could be using single quotes instead of double quotes.